src/public/drug/sections/MechanismsOfAction/Section.js

```diff
@@ -27,6 +32,10 @@ const columns = [
   {
     id: 'references',
     label: 'References',
+    filterValue: row =>
+      row.references.map(reference => reference.source).join(' '),
+    exportValue: row =>
+      row.references.map(reference => reference.source).join(),
     renderCell: d =>
```

Comment:
Just a thought/suggestion: we use `row =>` in all other cases above, while here we keep `d =>`. 
Could update it for consistency perhaps?

Reply:
Good point, I'll change it.

Reply:
Thanks @mirandaio - that's great